### ISSUE TITLE
Version 1.1+ broken

### ISSUE THREAD

**Issue by achimha** [radutopala/skype-bot-php#5]
The 1.1 code and the current master cannot send messages anymore. The token file is not picked up and no bearer is being sent. I can reproduce it on OS X and Linux.

**Comment by radutopala** [comment]
@achimha pls provide an API input and output example, with obfuscated token of course.

You can use `--debug` for the full output with headers and body.

**Comment by achimha** [comment]
I don't get much output:

```
$ vendor/radutopala/skype-bot-php/bin/skype --debug -vvv conversation:activity "8:myskypename" "a test message"
You should re-auth in the following 10 minutes.
Headers:
Array
(
    [Cache-Control] => Array
        (
            [0] => no-store, must-revalidate, no-cache
        )

    [Pragma] => Array
        (
            [0] => no-cache
        )

    [Content-Length] => Array
        (
            [0] => 0
        )

    [ContextId] => Array
        (
            [0] => tcid=7320352972384732152,server=DB5SCH101121811
        )

    [Date] => Array
        (
            [0] => Mon, 18 Jul 2016 17:00:52 GMT
        )

)
Body:

```

**Comment by radutopala** [comment]
Ah .. don't use `8:username` anymore .. you need to use the username's hash provided in the webhook. Each username now has a hash generated per bot, so the call should be like 

```
bin/skype --debug conversation:activity "29:<username_hash>"
```

See the V3 updates for more info and check your webhook dumps.

**Comment by achimha** [comment]
Looking at the code, both 1.1 and current master should use the v2 API. Also I didn't do this from a webhook but on the command line, following the sample in the README.md.

Can you confirm that 1.1 and master actually work with `8:<username>` as documented on the repo?

**Comment by radutopala** [comment]
`master` and `1.1` are using `v2/` endpoint, but the `<to>` param should contain `29:hash` instead of `8:username`. Skype doesn't allow sending to `8:username` anymore, in either `v2` of `v3`. 

And I'm still using `v2/` endpoints because `v3` endpoints are still not clearly defined in the docs, they say we should wait for an end of July final release.

**Comment by radutopala** [comment]
I've updated the README in the repo, please check.

**Comment by achimha** [comment]
Why does the 1.0 version work with `8:skypename`?

Also I can't get 1.1 to work with `29:usernamehash`, it behaves the same way as with the old syntax.

From my initial debugging it seems as if the 1.1 code does not pick up the bearer token with its request. To reproduce: simply checkout master on either OS X or Linux in a clean directory, do `composer update`, auth and then try to send a message.

**Comment by radutopala** [comment]
@achimha you were right, seems like the Symfony Yaml Component had a changed behavior on reading files. Fix in `1.1.1`, please test and see if it's fine now.

**Comment by achimha** [comment]
Yes, that fixes the problem and both the 8: and 29: syntax work. Thank you!
